code-experiments/src/transform_vars_permutation_helpers.c

```diff
@@ -53,10 +56,26 @@ static void coco_compute_random_permutation(size_t *P, long seed, size_t n) {
 static long coco_random_unif_int(long lower_bound, long upper_bound, coco_random_state_t *rng){
   long range;
   range = upper_bound - lower_bound + 1;
+  printf("%ld\n", ((long)(coco_random_uniform(rng) * (double) range)) + lower_bound);
   return ((long)(coco_random_uniform(rng) * (double) range)) + lower_bound;
 }
 
 
+/**
+ * @brief returns a uniformly distributed integer between lower_bound and upper_bound using seed
+ * without using coco_random_new.
+ */
+static long coco_random_unif_integer(long lower_bound, long upper_bound, long seed){
+  long range, rand_int;
+  double *tmp_uniform;
+  tmp_uniform=coco_allocate_vector(1);
+  bbob2009_unif(tmp_uniform, 1, seed);
+  range = upper_bound - lower_bound + 1;
+  rand_int = ((long)(tmp_uniform[0] * (double) range)) + lower_bound;
+  coco_free_memory(tmp_uniform);
+  return rand_int;
+}
+
```

Comment:
This function should probably be moved to a more prominent place because this is more general than just being a helper function of `transform_vars_permutation`. The question is: where shall it go? Most likely to `coco_random.c`, I guess?